src/NuGet.Clients/NuGet.Tools/NuGetTools.vsct

```diff
@@ -112,6 +112,16 @@
         </Strings>
       </Button>
 
+      <Button guid="guidDialogCmdSet" id="cmdidUpgradeProjectJsonNuGetProject" priority="0x0420" type="Button">
+        <Parent guid="guidReferenceContext" id="cmdAddReferenceGroup" />
```

Comment:
Wouldn't it make more sense to have this in the context menu for right clicking a project?

Reply:
I was following the pattern we had for packages.config (figured less controversy). 

I think we can consider adding it there (potentially in a future PR as I want *something* available asap.

cc @OliaG what do you think?

Reply:
I don't see how anyone would ever find this in VS unless they had right clicked a project.

Reply:
We'll use TN to get people to do it.
So they'll get a yellow bar.